bigquery2/Module.md

```diff
@@ -60,19 +60,64 @@ bigquery2:Client bigqueryClient = new(bigqueryConfig);
 ```
 
 
-The `listProjects` function lists all projects to which current user have been granted any project role.
+The `listProjects` remote function lists all projects to which current user have been granted any project role.
 ```ballerina
 // List projects.
-var projects = bigqueryClient->listProjects();
+var listProjectResponse = bigqueryClient->listProjects();
 ```
 
 The response from `listProjects` is a `ProjectList` object if the request was successful or an `error` on failure.
 
 ```ballerina
-    if (projects is bigquery2:ProjectList) {
-            io:print("Projects: ", projects);
-        } else {
-            io:println("Error: ", response);
-        }
+    if (listProjectResponse is bigquery2:ProjectList) {
+        io:print("Projects: ", listProjectResponse);
+    } else {
+        // Error will be printed.
```

Comment:
Shall we be consistent with these error messages? We've used " // Print the error." above.